### PR TITLE
Unify scripts for converting, quantizing and chatting

### PR DESCRIPTION
**Pull request by zhenwei-intel**
## Type of Change

feature

## Description

detail description 
JIRA ticket: [NLPTOOLKIU-818](https://jira.devtools.intel.com/browse/NLPTOOLKIU-818)

- [x] convert script
- [x] quantization script
- [x] chat script

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

### CODE REVIEW

**Review by zhentaoyu — APPROVED** [review]
LGTM

**Comment by airMeng** [comment]
do you need to remove useless scripts, ```convert_mpt.py``` etc

**Comment by zhenwei-intel** [comment]
> do you need to remove useless scripts, `convert_mpt.py` etc

convert_model.py will call convert_mpt.py when reading config model name is mpt

File: intel_extension_for_transformers/backends/neural_engine/graph/scripts/chat_llm.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        type=int,
+        help="number of tokens to keep from the initial prompt (default: 0, -1 = all)  ",
+        default=0,
+    )
+
+    args = parser.parse_args(args_in)
+    print(args)
+
+    path = Path(
+        Path(__file__).parent.absolute(), "../build/bin/main_{}".format(args.model_name)
```
Comment (zhentaoyu) [inline intel_extension_for_transformers/backends/neural_engine/graph/scripts/chat_llm.py:89]: #137 is merged. Could you please change the `main` to `chat`?

**Comment by airMeng** [comment]
> > do you need to remove useless scripts, `convert_mpt.py` etc
> 
> convert_model.py will call convert_mpt.py when reading config model name is mpt

please update https://github.com/intel/intel-extension-for-transformers/blob/64546a04cb9622a11a2388e0e3587d3de9e6f29f/.github/workflows/script/models/cpp_graph_inference.sh#L18 and internal validation repo as well

**Review by a32543254 — APPROVED** [review]
LGTM

**Comment by zhenwei-intel** [comment]
> > > do you need to remove useless scripts, `convert_mpt.py` etc
> > 
> > 
> > convert_model.py will call convert_mpt.py when reading config model name is mpt
> 
> please update
> 
> https://github.com/intel/intel-extension-for-transformers/blob/64546a04cb9622a11a2388e0e3587d3de9e6f29f/.github/workflows/script/models/cpp_graph_inference.sh#L18
> 
> and internal validation repo as well

The original validation methods still supports.

**Pull request merged**